prow/github/client.go

```diff
@@ -1484,6 +1484,21 @@ type githubError struct {
 	Message string `json:"message,omitempty"`
 }
 
+type labelRemover interface {
+	RemoveLabel(org, repo string, number int, label string) error
+}
+
+// RemoveLabel makes the RemoveLabel call and ignores a LabelNotFound error
+func RemoveLabel(c labelRemover, org, repo string, number int, label string) error {
```

Comment:
I think we should put this into the client, not a helper method.

Reply:
Add a bool on the client or something that toggles this behavior.

There's nowhere which doesn't use this logic, so making everyone now learn to call `github.RemoveLabel(ghc)` instead of `ghc.RemoveLabel()` is unnecessarily complicated.